clusterloader2/pkg/util/json.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+)
+
+// PrettyPrintJSON converts given data into formatted json.
+func PrettyPrintJSON(data interface{}) (string, error) {
```

Comment:
I suggest moving this into util.go file instead of having separate file for just this function (as I don't expect this to grow anyway).

Reply:
done